scanner/src/main/kotlin/LocalScanner.kt

```diff
@@ -520,3 +546,18 @@ private fun ScanResultContainer.deduplicateScanResults(): ScanResultContainer {
 
     return copy(results = deduplicatedResults)
 }
+
+/**
+ * Parse the given [versionStr] to a [Semver] object, trying to be failure tolerant.
+ */
+private fun parseVersion(versionStr: String?): Semver? =
+    versionStr?.let { Semver(normalizeVersion(it)) }
+
+/**
+ * Normalize the given [versionStr] to make sure that it can be parsed to a [Semver]. The [Semver] class
+ * requires that all components of a semantic version number are present. This function enables a more lenient
+ * style when declaring a version. So for instance, the user can just write "2", and this gets expanded to
+ * "2.0.0".
+ */
+private fun normalizeVersion(versionStr: String): String =
+    versionStr.takeIf { v -> v.count { it == '.' } >= 2 } ?: normalizeVersion("$versionStr.0")
```

Comment:
@pombredanne [announced](
https://ort-talk.slack.com/archives/C9NNJ54B1/p1603923131208200) that the next ScanCode release will have version 20.11, will we just call it 20.11.0 internally then to be Semver compatible?

Reply:
According to @pombredanne in Slack, [calver is semver compatible](https://ort-talk.slack.com/archives/C9NNJ54B1/p1603965438210800?thread_ts=1603923131.208200&cid=C9NNJ54B1).

Reply:
Yes, version numbers are internally expanded to at least 3 components to satisfy the Semver class. I hope, this will not create any ambiguities - assuming that the first patch release of ScanCode would then be 20.11.1.